main.nf

```diff
@@ -145,90 +174,85 @@ workflow {
         params.intersect_regions_index
         )
 
-    files_to_validate_ch = reference_ch
-        .mix(intersect_regions_ch)
-        .mix(tumor_input_chs.tumor_bam, tumor_input_chs.tumor_index)
-
-    if (params.samples_to_process.findAll{ it.sample_type == 'normal' }.size() > 0) {
-        files_to_validate_ch = files_to_validate_ch
-            .mix(normal_input_chs.normal_bam, normal_input_chs.normal_index)
-        }
+    if (params.input_type == 'bam') {
```

Comment:
Not necessary in this PR but the conditions here are getting complicated enough that I think having a separate workflow for each input type would be cleaner and easier to maintain rather than having a high level of nesting under conditions in a single workflow

Reply:
I made an issue